tools/gmhazard_scripts/db_creation/empirical_db/calc_emp_flt.py

```diff
@@ -37,6 +39,8 @@ def calculate_flt(
     suffix=None,
 ):
     nhm_data = sc.utils.flt_nhm_to_rup_df(nhm_ffp)
+    nhm_dict = nhm.load_nhm(nhm_ffp)
```

Comment:
I'd prefer to only load the NHM once rather than twice. Maybe the second is better for how the script is set up. But you could also do it as a dataframe with `load_nhm_df`